sourcegraph/sourcegraph
# insights: split query runner job into a separate worker job

closes #23053.

## Test plan
All worker jobs initialise correctly when running `sg start enterprise-codeinsights`. Then:
1. Create an insight that runs over all repos
2. Connect to the sg db (in my case, `psql -h localhost -p 5432 -U sourcegraph`)
3. Observe the query has been run to completion by the worker (`SELECT * from insights_query_runner_jobs;`)
